docs/en/enterprise-edition/policy-reference/aws-policies/aws-general-policies/bc-aws-272.adoc

```diff
@@ -36,24 +36,26 @@ This policy ensures that an AWS Lambda function has been properly configured to
 
 To address the issue, you need to enable the code-signing configuration for your AWS Lambda function. Code-signing adds an extra layer of security to your application by ensuring that the deployed code is not tampered with.
 
-[source,go]
+[source,terraform]
```

Comment:
```suggestion
[source,go]
```
Terraform is not supported, so we use `go` because it is the closest for formatting purposes.